internal/provider/vaultsecrets/resource_vault_secrets_secret.go

```diff
@@ -50,19 +50,23 @@ func (r *resourceVaultsecretsSecret) Schema(_ context.Context, _ resource.Schema
 				Description: "The name of the application the secret can be found in",
 				Required:    true,
 				Validators: []validator.String{
+					stringvalidator.LengthAtLeast(3),
```

Comment:
Apps use this common "hcp user settable slug":
https://github.com/hashicorp/cloud-vault-secrets/blob/main/internal/service/validation/service.go#L37

Wonder if it make sense to adopt this:
https://github.com/hashicorp/terraform-provider-hcp/blob/main/internal/providersdkv2/validators.go#L138

Reply:
Great find, did not know it exists! Thanks @VinnyHC